src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/MessageService.java

```diff
@@ -70,7 +70,8 @@ public String sendOrRespondMessage(final String profile, final Integer projectId
 
     // Creating RPA process input string
     var inputBuilder = RPAProcessInput.builder().userName(buyerUser.get().getEmail())
-        .password(rpaAPIConfig.getBuyerPwd()).ittCode(procurementEvent.getExternalReferenceId())
+        .password(rpaGenericService.getBuyerEncryptedPassword(buyerUser.get().getUserId()))
```

Comment:
Sonar is showing a warning here to check for user not being returned (as service returns an Optional):
`Optional value should only be accessed after calling isPresent() `

e.g. something like `var user = buyerUser.orElseThrow(() -> new IllegalStateException("blah.."));`